### PR TITLE
Add management command to republish course images 

### PR DESCRIPTION
**Pull request by schenedx**
This is the management command to republish course images from publisher to discovery courses

EDUCATOR-1761

@clintonb Please review

### CODE REVIEW

File: course_discovery/apps/publisher/management/commands/republish_course_images.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+class Command(BaseCommand):
+    help = 'Re-publish the course images we have on publisher to discovery course objects'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--start_id',
+            action='store',
+            dest='start_id',
+            default=None,
```
Comment (clintonb) [inline course_discovery/apps/publisher/management/commands/republish_course_images.py]: If the parameter is required, you don't need a default.

File: course_discovery/apps/publisher/management/commands/republish_course_images.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            required=True,
+            help='The Publisher course Primary key value starting id to re-publish the course image.'
+
+        )
+
+        parser.add_argument(
+            '--end_id',
+            action='store',
+            dest='end_id',
+            default=None,
```
Comment (clintonb) [inline course_discovery/apps/publisher/management/commands/republish_course_images.py]: Same note as above.

File: course_discovery/apps/publisher/management/commands/republish_course_images.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            required=True,
+            help='To this id of the Publisher Course the course image will be re-published.'
+        )
+
+    def handle(self, *args, **options):
+        """ Import the course according to the given range."""
```
Comment (clintonb) [inline course_discovery/apps/publisher/management/commands/republish_course_images.py]: This docstring isn't necessary.

File: course_discovery/apps/publisher/management/commands/republish_course_images.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        start_id = options.get('start_id')
+        end_id = options.get('end_id')
+
+        publisher_courses = Course.objects.filter(id__range=(start_id, end_id), image__isnull=False)
+        for publisher_course in publisher_courses:
+            discovery_course = None
+            try:
+                discovery_course = publisher_course.discovery_counterpart
+            except:  # pylint: disable=bare-except
```
Comment (clintonb) [inline course_discovery/apps/publisher/management/commands/republish_course_images.py]: Use the specific exception type.

File: course_discovery/apps/publisher/management/commands/tests/test_republish_course_images.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+from testfixtures import LogCapture
+
+from course_discovery.apps.core.tests.helpers import make_image_file
+from course_discovery.apps.course_metadata.tests.factories import CourseFactory as DiscoveryCourseFactory
+from course_discovery.apps.publisher.tests import factories
+from course_discovery.apps.publisher.management.commands.republish_course_images import logger as command_logger
+
+
+@ddt.ddt
+class RepublishCourseImageCommandTests(TestCase):
```
Comment (clintonb) [inline course_discovery/apps/publisher/management/commands/tests/test_republish_course_images.py]: Use the py.test format.

File: course_discovery/apps/publisher/management/commands/tests/test_republish_course_images.py [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+from django.test import TestCase
+from testfixtures import LogCapture
+
+from course_discovery.apps.core.tests.helpers import make_image_file
+from course_discovery.apps.course_metadata.tests.factories import CourseFactory as DiscoveryCourseFactory
+from course_discovery.apps.publisher.tests import factories
+from course_discovery.apps.publisher.management.commands.republish_course_images import logger as command_logger
+
+
+@ddt.ddt
```
Comment (clintonb) [inline course_discovery/apps/publisher/management/commands/tests/test_republish_course_images.py]: Use [py.test parameterization](https://docs.pytest.org/en/latest/example/parametrize.html).

File: course_discovery/apps/publisher/management/commands/tests/test_republish_course_images.py [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+        self.forganization_extension = factories.OrganizationExtensionFactory()
+        self.organization = self.forganization_extension.organization
+        self.publisher_course.organizations.add(self.organization)
+        self.command_name = 'republish_course_images'
+        self.command_args = [
+            '--start_id={}'.format(self.publisher_course.id),
+            '--end_id={}'.format(self.publisher_course.id)
+        ]
+
+    def _assert_discovery_course(self, discovery_course):
```
Comment (clintonb) [inline course_discovery/apps/publisher/management/commands/tests/test_republish_course_images.py]: `assert_images_exist`

**Comment by schenedx** [comment]
@feanil Please review

**Pull request merged**